.travis.yml

```diff
@@ -1,7 +1,8 @@
 language: node_js
 
 node_js:
-  - "6"
+  - "9"
+  - "10"
```

Comment:
Are you planning to support the latest LTS release of Node.js, i.e. 8? I would appreciate if that was supported and then in the test matrix too of course...

Reply:
Sure!